azure-kusto-data/azure/kusto/data/client_request_properties.py

```diff
@@ -9,6 +9,8 @@ class ClientRequestProperties:
     For more information please look at: https://docs.microsoft.com/en-us/azure/kusto/api/netfx/request-properties
     """
 
+    _CLIENT_REQUEST_ID = "Client Request ID"
```

Comment:
Do attributes traced usually have spaces in them?

Reply:
no - underscore and lowercase - will fix